### PR TITLE
install go in krte

### PR DESCRIPTION
**Pull request by BenTheElder**

### CODE REVIEW

File: images/krte/Dockerfile [hunk 1]
```diff
@@ -24,6 +24,7 @@ FROM debian:buster
 ARG IMAGE_ARG
 # arg that specifies the bazel version to install
 ARG BAZEL_VERSION_ARG=0.23.1
+ARG GO_VERSION
```
Comment (aojea) [inline images/krte/Dockerfile:27]: no default?

Comment (BenTheElder) [reply images/krte/Dockerfile:27]: shrug, it's supplied by the build.

File: images/krte/Dockerfile [hunk 2]
```diff
@@ -33,7 +34,8 @@ ARG BAZEL_VERSION_ARG=0.23.1
 # - hinting that we are in a docker container
 ENV IMAGE=${IMAGE_ARG} \
     BAZEL_VERSION=${BAZEL_VERSION_ARG} \
-    PATH=/google-cloud-sdk/bin:${PATH} \
+    GOPATH /home/prow/go \
```
Comment (BenTheElder) [inline images/krte/Dockerfile:37]: this requires an `=`
unfortunately nothing validates this in presubmit

Comment (BenTheElder) [reply images/krte/Dockerfile:37]: https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/post-test-infra-push-krte/1273674087943639041

**Pull request merged**